mgbuild/main.go

```diff
@@ -0,0 +1,41 @@
+package main
```

Comment:
Non top-level commands are usually put under the `cmd` subdirectory

Reply:
`go run go.einride.tech/mage-tools/cmd/build`

Reply:
:+1:

Reply:
Fixed